runtime/lib/apis/mqtt.js

```diff
@@ -7,9 +7,13 @@ const registry = require('./mqtt-registry').registry;
 const EventEmitter = require('events').EventEmitter;
 
 // FIXME(Yorkie): tweak to online?
-const endpoint = 'mqtt://mqtt-dev.rokid.com';
+// const endpoint = 'mqtt://mqtt-dev.rokid.com';
+const endpoint = 'mqtts:://wormhole.rokid.com:8885';
 let handle = null;
 
+// ca pem str
+const CA_PEM_STR = fs.readFileSync(__dirname + '/mqtt.pem');
```

Comment:
mqtt gateway is not using a self signed certificate now, the mqtt.pem is ueless.